### ISSUE TITLE
[ENHANCE] Energy system overhaul

### ISSUE THREAD

**Issue by AlienXtream** [DarkholmeTenk/TardisModIssues#468]
a new energy system that aims to increase the involvement and challenge of maintaining one's tardis WITHOUT the threat of total destruction.

New energy type, "rift" energy would fill the role of artron energy for flight. Rift energy reserves would NOT regenerate while in the time vortex and instead MUST be near a suitable rift (overworld near 0,0?) to refuel. 

Artron energy would be used to "power" various things inside the TT capsule such as roundels. all roundel components would have a passive energy cost in addition to the existing cost from nanogens and the restoration field. artron energy would ONLY recharge while in the time vortex at a rate determined by the speed of the tardis (energy regen upgrade would improve this as well as the rate the tardis refuels). this would also give more reason to use artron batteries to power roundels nearby

the room counter could be replaced by an artron cost. the larger the room the more artron energy it would  take and the more rooms you have the higher the cost modifier (1 room = x1 2 rooms = 1.1 for example?)the current max rooms upgrade could change to room cost, higher levels reduce room cost and multiplier

**Comment by AlienXtream** [comment]
i swear there were atleast 2 comments here O.o?!

**Comment by SomeBadGamer** [comment]
Thare was i was one of them

**Comment by AlienXtream** [comment]
Lol xD

**Comment by SomeBadGamer** [comment]
I like the idea

**Comment by AlienXtream** [comment]
As do i but it is not down to us. Look at it this way, dark hasn't said no :p. And believe me, he WILL have seen the post by now

**Comment by ghost** [comment]
Yes and this is not resource reliant meaning it is possible to add.

**Comment by FoxPotato** [comment]
I mean the resource reliable option was also possible, it would just get OP :P

**Comment by Webber88** [comment]
Well how about adding it as a placeable thing for people who are careless so they can go into creative and recharge it. And maybe reduce your proposed cost for rooms by .5 since your proposed cost just seems a little high. But other than that I like the idea. :)

**Comment by DarkholmeTenk** [comment]
#88 similar to this?
It'll probably be a property of the world, not a specific block or anything and then the rendering only needs to be stuck on to the TTE renderer.

**Comment by AlienXtream** [comment]
yeh

**Comment by ghost** [comment]
Ohoh, who's up for the canon Rift being a block? I'd love to see that!
For those who don't know what I'm referring to, in Doctor Who, there's a Time Rift in Cardiff Bay, which is around a waterfall. I live not too far from that thing, so I'd love to see it in game, if we move to Rift energy!

**Comment by FoxPotato** [comment]
if rift energy would be implemented it wouldn't come in the form of the cardiff waterfall statue thing lol 
since it's a TARDIS mod and not a doctor who mod, and especially not a torchwood mod ;P
it would most likely be an area, or perhaps something like aura nodes in thaumcraft. You could always build a structure around it ;p
